scripts/InstancingWithModsSystem.js

```diff
@@ -0,0 +1,129 @@
+class InstancingSystem extends System {
+
+    // todo - title it InstancingWithModsSystem
+
+    constructor() {
+        super();
+
+        console.log(this.componentName);
+
+        const this.instanceCount = 5;
+
+        // DataOrientedDesign arrays
+        let this.positions = null;
+        let this.scales = null;
+        let this.rotations = null;
+        let this.meshes = [];
+        let this.geometry = null;
```

Comment:
Remove the const and lets in front of anything with `this.` in front of it 😬

Reply:
using `this` defines a property on the class. so no need for var, const, let, or function

Reply:
√